hAMRonization/ResFinderIO.py

```diff
@@ -37,51 +41,94 @@ def __init__(self, source, metadata):
             # parsed from top level of within class results
             '_gene_name': 'gene_name'
         }
+        except(ValueError): # Resfinder 4 tsv file
```

Comment:
Is there a way to make this a positive filter rather than an exception failover i.e. if it has a field that is only in resfinderv4 like phenotype use that mapping and if it has the input filename use the resfinder3 mapping.

I'm always a bit cautious using catching exceptions as part of control flow, as it can capture ValueError for more reasons than just the intended failover.  Given the worryingly common malformed data, it would be nice for errors to be thrown for them as expected.